src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -86,34 +86,37 @@ public class UserManagementService : IUserManagementService
                 });
         }
 
-        public async Task<Either<ActionResult, UserReleaseRoleViewModel>> AddUserReleaseRole(Guid userId,
+        public async Task<Either<ActionResult, Unit>> AddUserReleaseRole(Guid userId,
             UserReleaseRoleRequest userReleaseRole)
         {
             return await _userService
                 .CheckCanManageAllUsers()
                 .OnSuccess(async () =>
                 {
                     return await _persistenceHelper
-                        .CheckEntityExists<Release>(userReleaseRole.ReleaseId)
-                        .OnSuccess(() => ValidateUserReleaseRoleCanBeAdded(userId, userReleaseRole))
-                        .OnSuccess(async () =>
+                        .CheckEntityExists<Release>(
+                            userReleaseRole.ReleaseId,
+                            q => q.Include(r => r.Publication)
+                        )
+                        .OnSuccess(release =>
                         {
-                            var newReleaseRole = new UserReleaseRole
-                            {
-                                ReleaseId = userReleaseRole.ReleaseId,
-                                Role = userReleaseRole.ReleaseRole,
-                                UserId = userId
-                            };
-
-                            await _contentDbContext.AddAsync(newReleaseRole);
-                            await _contentDbContext.SaveChangesAsync();
-
-                            var response = await GetUserReleaseRole(newReleaseRole.Id);
```

Comment:
The intention for doing this was to get rid of `GetUserReleaseRole` which can be removed now :)

Reply:
Good point :)